golang/internal/cache.go

```diff
@@ -561,62 +561,6 @@ func StoreAssetIDToCache(customerID string, location string, assetID string, DBa
 
 	b := strconv.Itoa(DBassetID)
 
-	err := rdb.Set(ctx, key, b, dataExpiration).Err()
-	if err != nil {
-		zap.S().Errorf("redis failed")
-		return
-	}
-}
-
```

Comment:
I didn't delete this part, I just deleted the caching parts. Is it just a displaying wrongly or some underlying issue?
Please check @JeremyTheocharis.

Reply:
The red marking seems to be 6 lines to high.